Problem1.java

```diff
@@ -0,0 +1,87 @@
+/*
+construct a adjacency list which consits of all the possible combinations of the words
+in the wordlist. 
+ex: *it, h*t, etc
+Start the bfs from begin word by adding it to queue
+and for each possible combination of the word, fetch the adjacency list from map and chck if its the endword, return the count. Else continue by adding node to the queue.
+time complexity: O(n^2) + O(m^2)
+space complexity: O(m^2 n)
+*/
+class pair{
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).